### PR TITLE
Roboticists

### PR DESCRIPTION
**Pull request by KingFroozy**
## About the PR

Roboticists coming to the game, introducing their own Job, Bags, PDAs and ID cards, Airlocks, Lockers, and access.
Job Spawn is included, I put my own character there as a reference :3

Roboticists of coure have a job description, as well as all their important stuff. Also the science department's description was slightly changed due to addition of roboticists

I also changed the Robodrobe's and Robotech's deluxe access to  Robotics, and Robodrobe was slightly resprited to match the Robotics style, also adjusted it's inventory to match the Roboticist's clothes.

Roboticists, Senior Researcher, Research Director, Head of Personnel, and everyone with AA access have access to robotics.

Robotics airlock are: Standard, Standard locked, Glass, Glass locked, Maintenance locked, and secure windoor locked. These airlocks are available to be chosen in spray painter's UI.
Robotics access is visible in the ID card computer and access configurator.

Their lockers contain: A pair of robohands gloves, gasmask, a coat, a headset, a welding mask and a filled utility belt.
Robotics wardrobe's inventory was changed to match the Roboticist's clothes

WARNING: Roboticists are not mapped on any station yet! Any assistance with mapping would be very welcome.

## Why / Balance

Only two (2) roboticists are available to play at the station (As I believe, it should be), because their job isn't that important, but not that useless to have only one roboticist.

Robotics lockers contain a welding mask and a filled utility belt due to many mechs and cyborgs require having a welder and a bunch of tools to complete them.
Roboticist's starter bags contain a standard survival box in it.

To play as roboticist, you have to play in the science department for at least 6 hours.

## Technical details

Content.Shared / Access / components / IdCardComputerComponent.cs was changed to make the robotics access appear in the ID card computer's UI

## Media

Снимок экрана 2023-08-21 183235

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added Roboticists. Construct various mechs, borgs and robots to help the station crew. They have their own bags, access, PDAs and ID cards, lockers, and airlocks. Not mapped yet!
- add: Robotics bags were added to the game. You can find them in the Robodrobe.
- add: Robotics access is now available in the ID card computer and access configurator.
- tweak: Robotech deluxe and Robodrobe now have "Robotics" access, Robodrobe's inventory and sprites were slightly adjusted to match the roboticist's style.
- tweak: Robotics wardrobe's inventory was adjusted to match the roboticist's clothes.
- tweak: Science department's description was sligthly changed

### CODE REVIEW

File: Resources/Locale/en-US/job/department-desc.ftl [hunk 1]
```diff
@@ -4,5 +4,5 @@ department-Command-description = Manage the crew and keep them working efficient
 department-Engineering-description = Keep the station operational.
 department-Medical-description = Keep the crew healthy.
 department-Security-description = Keep the peace around the station.
-department-Science-description = Research new technologies and dangerous artifacts.
+department-Science-description = Research new technologies, dangerous artifacts, and build robots.
```
Comment (deltanedas) [inline Resources/Locale/en-US/job/department-desc.ftl:7]: , and
should be
and

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
also why should senior researcher not have it, senior engi has atmos and senior phys has chem

**Comment by KingFroozy** [comment]
> also why should senior researcher not have it, senior engi has atmos and senior phys has chem

I changed the thing, since roboticists aren't playable yet. They have to be mapped to require to play as them for the other roles, else it would be impossible to play as Senior Researcher or RD

**Comment by deltanedas** [comment]
as in access to the doors, not a playtime required

**Comment by KingFroozy** [comment]
> as in access to the doors, not a playtime requirement

Oh, wait, you're right

**Comment by EmoGarbage404** [comment]
Calling a veto on a robotics job.

Quite plainly, robotics is so integrated into the general of science that it doesn't make sense to section it off from the standard job. Separating what few robotics mechanics we have into a separate job when that very same job is forced to do regular scientist work 90% of the time seems rather pointless.

Furthermore, unlike something like atmospherics or chemistry, where there are higher stakes and greater complexity, robotics isn't really something that warrants being more selective with who can interact with it. If anything, this is simply preventing people from engaging with interesting and fun mechanics.

I don't really see a reason to add this job into the game and you haven't really justified it in your PR description either.

Closing this pr.

**Comment by EmoGarbage404** [comment]
Reopening just to get some second opinions.

you're still free to add the clothes and such into the robodrobe.

**Comment by deltanedas** [comment]
giving borg panels robotics access reader might make it important enough, since then you need to kill a roboticist or emag it twice, neither are ideal for tators

**Comment by HoidC** [comment]
I want to say Roboticists would be a good role if robots and modules were cheaper but as it is, they don’t really get much until later into the round. Silver isn’t mapped into all vaults and can only be obtained through Salvage, of which isn’t always (ever) reliable, and borgs need a LOT of steel to make until you get the machine upgraded.

If there was more early round bot content maybe roboticist would be good but until then I personally don’t see the point

**Comment by deltanedas** [comment]
1. make vims and hamtr
2. salv died and got replaced by pros so silver is on the menu
3. hampter helps make borgs

**Comment by HoidC** [comment]
> * make vims and hamtr
> * salv died and got replaced by pros so silver is on the menu
> * hampter helps make borgs

deltanedas be comprehensible challenge (impossible difficulty)[gone wrong]{gone professional}

**Comment by Nimfar11** [comment]
Theoretically, a roboticist could be a useful role, but the problem is that there's nowhere to use it right now. The Borg is in its infancy and doesn't have half of what it was designed for, including the AI itself. Robotics has nothing to do yet. I rather think that this role is premature. Later, perhaps, but now it seems too early.

**Comment by Endecc** [comment]
I agree with the feedback of how Cyborgs, and as a whole Robotics, are still very young. Unless some argument could be made if you also consider a roundstart Borg, but even then that would amount to "Babysit this 1 robot until you can make more in 20 minutes."

I want to say I would love this role, it was one of my favorites when I was very active in SS13. But there really isn't much for it to do in the current state of Robotics.

I think it's worth holding onto as a role for later, though. Integrating the clothing vendor and the outfits for Scientists who want to play the part, I also think is a great idea. But it's just not fun enough for players to be locked in that robotics lab. Not quite yet.

**Comment by UbaserB** [comment]
In the first place, to balance the job and borging i feel creating the basic borgs should be available (no modules, or maybe one or two basic types) roundstart. If this were to happen there could also be a seperate robotics research tree which slowly lets you unlock the different types and modules, for example janitor bots at first, and then engineer borgs at the end. This would allow robotics to keep working throughout the whole round, and scientists to actually have a job (for a while at least).

**Comment by potato1234x** [comment]
i like this, however the drobe sprite change looks a bit out of place on a normal map

**Comment by potato1234x** [comment]
i'd definitely make the color scheme purple and black like the robodrobe

**Comment by basyachok** [comment]
i was a roboticist main when i was playing ss13.
yep, this pr is completely pointless at the current time, it will also cause more conflicts and harm to the RND rather than bring something useful, imo we don't actually have the robotics at all.
i can't imagine a roboticist gameplay without being able to build a lethal mecha

too early

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Ideally this just becomes a loadout for scientists as there's no reaosn to have it as an entirely separate role as robotics doesn't need its own access.

Atmos and chem there's at least a highly volatile department that needs access-gating.

**Comment by Froozigiusz** [comment]
I think that for the roboticist to work, simple robots would need to get a refactor alongside maintenance drone (an intermediate level robot alongside pAI?). Right now many robotic projects are found within the construction menu, which funny thing, cyborgs can also create them (and drones in the lathe, if they have tool module active). Drones being in the lathe is also incredibly confusing for a robot that it is.
Maybe a roboticist shuld be more of a shepperd, controlling the flow of all types of robots on the station. Having access to the status computer would also allow to disable some simpler robots and hamper bigger ones, and maybe even having access to the compiler/binary radio channel.

**Comment by Hmeister-fake** [comment]
I think it's too early to add robo. At the moment science is basically 4 things, research, anomalys, artifacts and robotics. Removing one of these for a seperate job with very minimal content that still relys on science researching robo modules seems silly. When the medical refactor is done this would be great to PR with Robo surgery because then there would be a good reason for it to be a seperate job, since you would need surgery experience and science experience to play it.

**Pull request closed without merging**